src/scripts/main.js

```diff
@@ -1,3 +1,206 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+let sortDirection = true;
+
+// 1. Implement table sorting by clicking on the title (in two directions).
+function sortTable(columnIndex) {
+  const rows = [...table.querySelectorAll('tbody tr')];
+
+  rows.sort((rowA, rowB) => {
+    let cellA = rowA.children[columnIndex].innerText;
+    let cellB = rowB.children[columnIndex].innerText;
+
+    if (columnIndex === 3) {
+      cellA = parseInt(cellA);
+      cellB = parseInt(cellB);
+    }
+
+    if (columnIndex === 4) {
+      cellA = parseFloat(cellA.slice(1).split(',').join(''));
+      cellB = parseFloat(cellB.slice(1).split(',').join(''));
+    }
+
+    if (sortDirection) {
+      return cellA > cellB ? 1 : -1;
+    }
+
+    return cellA > cellB ? -1 : 1;
+  });
+
+  rows.forEach((row) => table.querySelector('tbody').appendChild(row));
+  sortDirection = !sortDirection;
+}
```

Comment:
The sortTable function seems to be sorting the table directly in the DOM. This is not a good practice because manipulating the DOM is slow. It would be better to sort the data in JavaScript and then render it.